packages/nc-gui/components/smartsheet/column/QrCodeOptions.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+import { UITypes } from 'nocodb-sdk'
+import type { SelectProps } from 'ant-design-vue'
+import { useVModel } from '#imports'
+
+const props = defineProps<{
+  value: any
+}>()
+const emit = defineEmits(['update:value'])
```

Comment:
No idea who thought that using `v-model:value` would be the right way to go but I suggest just using the regular v-model and not using a different prop name for it.

```ts
interface Props {
   modelValue: any
}

const props = defineProps<Props>()

const emit = defineEmits(['update:modelValue'])

const vModel = useVModel(props, 'modelValue', emit)
```

Then you can bind the value like this

```vue
<!-- v-model="formState" instead of v-model:value="formState" -->
<LazySmartsheetColumnQrCodeOptions v-if="formState.uidt === UITypes.QrCode" v-model="formState" />
```